docs/05_advanced.md

```diff
@@ -306,7 +306,44 @@ making some changes to the repository at `/cvmfs/repo.organization.tld`, and the
 cvmfs_server publish repo.organization.tld
 ```
 
-## 5.5 Using a configuration repository
+## 5.5 Mounting CernVM-FS repositories as an unprivileged user
+
+The default way of installing and configuring the CernVM-FS client requires you to have root privileges.
+In case you want to use CernVM-FS repositories on systems where you do not have these, there are still some ways to install the client and mount repositories.
+We will show two different methods: using a [Singularity](https://sylabs.io/singularity/) container, and [cvmfsexec](https://github.com/cvmfs/cvmfsexec).
+
+### 5.5.1 Singularity
+
+Recent versions of Singularity offer a `--fusemount` option that allow you to mount CernVM-FS repositories.
+In order for this to work, you will need to install the `cvmfs` and `cvmfs-fuse3` package inside your container,
+and add the right configuration files and public keys for the repositories.
+Furthermore, you need two make two directories on the host system that will store the CernVM-FS cache and sockets;
+these need to be made available via a bind mount inside the container at `/var/lib/cvmfs` and `/var/run/cvmfs`, respectively.
+
+As an example, you can run the [EESSI pilot client container](https://eessi.github.io/docs/pilot/#accessing-the-eessi-pilot-repository-through-singularity) (which was built using [this Dockerfile](https://github.com/EESSI/filesystem-layer/blob/master/containers/Dockerfile.EESSI-client-pilot-centos7-x86_64)) using Singularity by doing:
+```
+mkdir -p /tmp/$USER/{var-lib-cvmfs,var-run-cvmfs}
+export SINGULARITY_BIND="/tmp/$USER/var-run-cvmfs:/var/run/cvmfs,/tmp/$USER/var-lib-cvmfs:/var/lib/cvmfs"
+export EESSI_CONFIG="container:cvmfs2 cvmfs-config.eessi-hpc.org /cvmfs/cvmfs-config.eessi-hpc.org"
+export EESSI_PILOT="container:cvmfs2 pilot.eessi-hpc.org /cvmfs/pilot.eessi-hpc.org"
+singularity shell --fusemount "$EESSI_CONFIG" --fusemount "$EESSI_PILOT" docker://eessi/client-pilot:centos7-$(uname -m)
+```
+
+Note that you have to be careful when launching multiple containers on the same machine:
+in this case, they all need a separate location for the cache, as it cannot be shared across containers.
+
```

Comment:
You might want to refer to  [singcvmfs](https://github.com/cvmfs/cvmfsexec#singcvmfs-command) to make using singularity --fusemount easier.

Reply:
Thanks, added `singcvmfs` to the section about `cvmfsexec`.